jax/experimental/jax2tf/README.md

```diff
@@ -84,3 +84,7 @@ To run jax2tf on GPU, both jaxlib and TensorFlow must be installed with support
 for CUDA. One must be mindful to install a version of CUDA that is compatible
 with both [jaxlib](../../../../../#pip-installation) and
 [TensorFlow](https://www.tensorflow.org/install/source#tested_build_configurations).
+
+## Known limitations
+
+
```

Comment:
Did you forget to add documentation on this behaviour here in the README?

Reply:
:-) I was planning to add it in a second commit while the first one was running the google tests. Done.